src/components/progressTracker/theme.css

```diff
@@ -13,7 +13,8 @@
   display: flex;
   flex-direction: row;
   justify-content: center;
-  padding: 20px 24px;
+  padding: 20px 30px;
+  max-width: 1600px;
```

Comment:
The max-width seems to be unnecessary to me. If you want to control the max-width, you can do it with a wrapper div.

Reply:
You're right, I'll remove it 👍